backend/resolvers/types/showType.js

```diff
@@ -2,6 +2,9 @@ export default {
   Show: {
     judges (show) {
       return show.getUsers()
+    },
+    entries (show) {
+      return show.getEntries()
```

Comment:
Here is where we can get interesting..... I believe we can do something like the following, which prevents us from leaking other peoples' entries to the front-end.

```javascript
entries (show, _, req) {
  if (req.auth.type === ADMIN || req.auth.type === JUDGE)
    return show.getEntries()
  else
    // return only this user's relevant entries
    return User.findById(req.auth.username)
      .then(u => u.getOwnAndGroupEntries())
      .then(entries => entries.filter(e => e.showId === show.id)
}

// NOTE THAT THIS IS JUST AN EXAMPLE:
// I bet there's a neat sequelize way to do this without
// throwing out unneeded shows, but whatever this
// is probably fast enough
```